test/verify/check-docker-storage

```diff
@@ -84,19 +85,22 @@ class TestDockerStorage(MachineCase):
             b.wait_present("#containers-storage-details a")
 
     def testDevmapper(self, vg=""):
-        b = self.browser
+        m = self.machine
 
         # On the Atomics, we use two machines: one for running
         # cockpit-ws, and one whose docker storage pool is managed.
         # We can't do that both on a single machine since resetting
         # the pool would kill the cockpit-ws container.
 
         if self.machine.atomic_image:
-            m = self.new_machine(machine_key='additional')
-            m.start()
-            m.wait_boot()
+            login_machine = self.new_machine(machine_key='login',
+                                             image=get_build_image(self.machine.image))
```

Comment:
I think we should have a comment here explaining why we use the `build` image as the `login` machine. For example:

```
# We want to use a non-Atomic machine as the login machine
# and the build machine that matches the current Atomic is suitable
```